File: Application/WebAPI/Data/Models/Review.cs
namespace WebAPI.Data.Models
{
    public class Review : Entity
    {
        public int Status_Id { get; set; } 
        public int User_Id { get; set; } 
        public int UserReceiving_Id { get; set; } 
        public int Object_Id { get; set; } 
        public int EntityType_Id { get; set; } 
        public int Order_Id { get; set; } 
        public int OrderPackage_Id { get; set; } 
        public int Rating { get; set; } 
        public string Title { get; set; } 
        public string Content { get; set; } 
        public bool IsComplaint { get; set; } 
        public bool IsResolved { get; set; } 
    }
}

File: Application/WebApp/src/app/models/review.ts
import { Entity } from './Entity';

export interface Review extends Entity {
  status: string;
  user: number;
  userReceiving: number;
  object: number;
  entityType: string;
  order: number;
  orderPackage: number;
  rating: number;
  title: string;
  content: string;
  isComplaint: boolean;
  isResolved: boolean;
}
